libs/external-router/guides/drivers/testing.md

```diff
@@ -0,0 +1,14 @@
+# `@daffodil/external-router/driver/testing`
+
+When working with external services, you often want to run tests without depending on the external system's relability or performance. As you write tests for features that depend upon the `@daffodil/external-router/driver` you will find yourself looking for a simple way to meet the `DaffExternalRouterDriverInterface` interface requirements without over-complicating your tests. In these scenarios, having a testing driver that emulates the behavior of an external service is extremely useful. The `DaffExternalRouterTestingDriver` is well-suited for these purposes.
+
+The `DaffExternalRouterTestingDriver` of the `@daffodil/external-router` is useful for Unit Tests, Integration Tests, and simple Acceptance Tests where you don't want to depend on an external service over HTTP. The driver is intended to be:
```

Comment:
I think this should say that the `DaffExternalRouterTestingDriver` exports from `@daffodil/external-router/driver/testing` subpackage, just to be clear

Reply:
https://github.com/graycoreio/daffodil/pull/1240/commits/e84f901398d30c4af371fb257714725339f3640a